munet/native.py

```diff
@@ -1473,7 +1490,11 @@ async def run_cmd(self):
                     # u'--entrypoint=""',
                     f"--volume={shebang_cmdpath}:/tmp/{script_name}.shebang",
                 ]
-
+        if self.config.get("cmd-file", "").strip():
+            self.logger.critical(
```

Comment:
Raise an exception here, we shouldn't ignore this error it should stop until the user addresses the issue.